packages/core/src/theme/theme.ts

```diff
@@ -232,23 +232,71 @@ export const textShadows = {
   md: `0 2px 4px rgba(0,0,0,0.5)`,
 }
 
+// opacity
+const opacity = {
+  'opacity-0': 0.0,
+  'opacity-10': 0.1,
+  'opacity-20': 0.2,
+  'opacity-30': 0.3,
+  'opacity-40': 0.4,
+  'opacity-50': 0.5,
+  'opacity-60': 0.6,
+  'opacity-70': 0.7,
+  'opacity-80': 0.8,
+  'opacity-90': 0.9,
+  'opacity-100': 1.0,
+}
+
+// scale (min/half/max)
+const scale =  {
+  'scale-0': 0.0,
+  'scale-10': 0.1,
+  'scale-20': 0.2,
+  'scale-30': 0.3,
```

Comment:
@wesleymartin85 can you confirm that these are correct, or should they be `opacity`?